app/scripts/lib/open-update-tab-and-reload.test.ts

```diff
@@ -0,0 +1,43 @@
+import browser from 'webextension-polyfill';
+import { openUpdateTabAndReload } from './open-update-tab-and-reload';
+
+jest.mock('webextension-polyfill', () => ({
+  tabs: {
+    create: jest.fn(),
+  },
+}));
+
+describe('openUpdateTabAndReload', () => {
+  const mockRequestSafeReload = jest.fn();
+
+  beforeEach(() => {
+    jest.clearAllMocks();
```

Comment:
Generally we should be using `resetAllMocks` over `clearAllMocks` - it does the same as clear but _also_ resets mock implementations, helping to avoid accidentally influencing one test from another. We have a ticket for enabling `resetMocks` in the Jest config, but we've been working towards that slowly by manually adding `resetAllMocks` to each test suite.

But w/e, we can deal with this in a later PR, and we already have a ticket tracking this anyway. Just mentioning for future reference.